go/obscuronode/host/host.go

```diff
@@ -129,23 +129,28 @@
 
 // Start initializes the main loop of the node
 func (a *Node) Start() {
+	log.Log(fmt.Sprintf("Waiting for enclave - %v", a.ID))
```

Comment:
The `waitForEnclave` will log something similar, is this needed ?

Reply:
Yeah good shout, removed (and tidied up the other logs I left lying around here - was using a very manual remote debugging method at the time haha).